src/sentry/utils/snuba_rpc.py

```diff
@@ -99,24 +103,31 @@ def rpc(req: SnubaRPCRequest, resp_type: type[RPCResponseType]) -> RPCResponseTy
 
 
 def _make_rpc_request(
-    endpoint_name: str, class_version: str, req: SnubaRPCRequest
+    endpoint_name: str,
+    class_version: str,
+    req: SnubaRPCRequest | CreateSubscriptionRequest,
```

Comment:
Currently, the `CreateSubscriptionRequest` protobuf doesn't have a `meta` attribute with a `referrer`. I've added `CreateSubscriptionRequest` as a valid type here to handle these scenarios. I'm open to updating the protobuf to include `meta` instead if we feel that would be better

Reply:
I think this is fine for now, I'll clean this up in a future pass :+1: